src/utils/dayViewLayout.js

```diff
@@ -328,3 +328,33 @@ export default function getStyledEvents ({
 
   return styledEvents
 }
+
+/**
+ * Takes an array of availabilies and returns an array
+ * containing the same availabilies, but with an additional style property.
+ *
+ */
+export function getStyledAvailabilities ({
+  availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin, date
+}) {
+  let styledAvailabilities = [];
+
+  if (!availabilities) return styledAvailabilities;
+
+  availabilities.forEach((availability) => {
+    const startTime = availabilityStartAccessor({availability, date});
```

Comment:
Why do these start and end accessors need `{ availability, date }` and not just `availability`? I would have expected them to be called like `availabilityStartAccessor(availability)`.